README.md

```diff
@@ -30,132 +82,111 @@ Entry point is at `frontend/src/index.js`.
-
-
-
-Install virtualenv:
-`$ pip install virtualenv`
-
-
-# Application Download & Installation
+#### SMTP (Emailing)
+
+The BEAR-Necessities-Market application uses [sendgrid][sendgrid] for our smtp server and customer engagement platform.
+
+
+#### Deployment
+
+The BEAR-Necessities-Market application uses [Heroku][heroku] to deploy, manage, and scale quickly and cost efficiently
+
+
+## Code Structure
+
+```bash
+.
+├── CHANGELOG.md
+│      // Application releasing log
+├── LICENSE
+│      // Platform license
+├── README.md
+│      // Initial entry point for documentation
+├── backend
+│   ├── app.py
+│   │      // Flask app definition
+│   ├── commands.py
+│   │      // Extra flask commands declarations
+│   ├── config.py
+│   │      // Database/application config
+│   ├── extensions.py
+│   │      // Module for declaring flask extensions (e.g. JWT)
+│   ├── models
+│   │      // Flask models (ORM) python classes for database mapping
+│   ├── resources
+│   │      // RESTful Routes
+│   └── utils
+│          // Custom defined utilities files
+├── docs
+│      // Documentation files
+│
+├── frontend
+│   ├── build
+│   │      // Production build, entrypoint at build/index.html
+│   │   
+│   ├── package.json
+│   │      // React dependencies list
+│
+│   ├── public
+│   │   ├── index.html
+│   │   │      // React application entrypoint at id="root"
+│   ├── src
+│   │   ├── _actions
+│   │   │      // dispatch server calls and call reducers
+│   │   ├── _constants
+│   │   │      // Stylesheets & action types
+│   │   ├── _helpers
+│   │   │      // Reusable frontend helper methods
+│   │   ├── _reducers
+│   │   │      // State modifiers
+│   │   ├── _services
+│   │   │      // API calls (http requests)
+│   │   ├── _store
+│   │   │      // Application state tree
+│   │   ├── components
+│   │   │      // Core reusable components
+│   │   ├── containers
+│   │   │      // Main application components 
+│   │   ├── layouts
+│   │   │      // Application layouts
+│   ├── tests
+│          // Frontend e2e tests
+│
+├── manage.py
+│      // Flask app entry point
+│
+├── media
+│      // media file storage
+│
+├── migrations
+│      // Database migration tracking
+│
+├── requirements.txt
+│      // backend application dependencies list
+│
+├── tests
+│      // Backend unit & functional tests
+│
+└── wsgi.py
+       // Production backend entrypoint
 
 ```
-$ git clone https://github.com/LibreFoodPantry/BEAR-Necessities-Market.git
-$ cd BEAR-Necessities-Market
-$ virtualenv env --python=python3.6
-
-ACTIVATE_MAC_ENV:
-$ source env/bin/activate
-
-ACTIVATE_WINDOWS_ENV:
-$ cd env/Scripts
-$ ./activate
 
---> Install server dependencies
-$ pip install -r requirements.txt --no-cache-dir
 
---> Tell Flask to run the manage.py with $ flask
-$ export FLASK_APP=manage.py
+### Discussion
 
---> Run db migrations
-$ flask db upgrade
+Say hello in our [slack channel](https://librefoodpantry.slack.com/messages/CG71A9VT2/)
```

Comment:
In the last LibreFoodPantry meeting, to avoid spam bots, it was agreed that we would not publicize the slack join URL. Please remove this.

We do need to figure out how folks should contact us. How do other FOSS projects do this without getting spammed?

Reply:
librefoodpantry@googlegroups.com

Reply:
Completed in next push.